CarInspection/src/WebCarInspection/wwwroot/css/bootstrap-datetimepicker.css

```diff
@@ -0,0 +1,418 @@
+/*!
```

Comment:
It should be fetched from lib folder (it's external component).

Reply:
Of course, `_Layout.cshtml` should be fixed.

Reply:
This is a specific datetimepicker (not library) so it should be in the folder css